types/utils.go

```diff
@@ -66,7 +66,10 @@ func reloadConfigurationFileIfNeeded(configFile string, storeOptions *StoreOptio
 		return
 	}
 
-	ReloadConfigurationFile(configFile, storeOptions)
+	if err := ReloadConfigurationFile(configFile, storeOptions); err != nil {
+		logrus.Warningf("Failed to reload %s %v\n", configFile, err.Error())
```

Comment:
```suggestion
		logrus.Warningf("Failed to reload %q %v\n", configFile, err.Error())
```
to be very sure the output is unambiguous with weird paths.

---

(`.Error()` is implied on `fmt` formatting and does not need to be spelled out, but also doesn’t hurt.)